BDB Agena Configs

SkyPhoenix999 opened the pull request:
RO Configs for Agena from BDB

SkyPhoenix999 committed:
Agena Configs

SkyPhoenix999 committed:
Updated Temp Values

SkyPhoenix999 committed:
Updated Temps

SkyPhoenix999 committed:
Fat Trimming

SkyPhoenix999 committed:
Pasted the wrong file

SkyPhoenix999 committed:
Fat Trimming

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+		volume = 30
+		basemass = -1
+	}
+	%useRcsConfig = RCSBlockTenth
```
Too weak (and probably too light), this was a spare Gemini Reentry Control system, which was fitted with 25 lbf thrusters

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+	@title = Agena Target Vehicle Docking Port
+	@manufacturer = McDonnell Aircraft
+	@description = This special docking port is designed to capture the Gemini nose cone's docking mechanism. The simple mechanical connection won't even allow you to transfer resources, much less crew, but it will securely attach two vessels together, for berthing or travel.
+	@mass = 0.05
```
This should be 211 kg to match ROCapsules

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+	@attachRules = 1,1,1,1,0
+	@MODULE[ModuleDockingNode]
+	{
+		@nodeType = Gemini
```
This should be GeminiAgena to match ROCapsules

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+	%rescaleFactor = 1.6
+	@title = Agena Target Vehicle Running Light
+	@manufacturer = North American Rockwell
+	@description = Essentially a pen light on the end of a straw, this light (traditionally mounted to the decoupler ring on the Apollo CSM) swings out to provide illumination to aid kerbonauts on EVAs.
```
Did you put the wrong description here?

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+			{
+				IDENTIFIER { name = ModuleFuelTanks }
+				DATA
+				{
```
Should be about 30 liters per tank.

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+		amount = 1500
+		maxAmount = 1500
+	}
+	@MODULE[ModuleRCSFX]
```
RCS ISP is too high for HTP

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena_Engines.cfg:
```diff
@@ -0,0 +1,335 @@
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
```
You should use this nozzle config

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena_Engines.cfg:
```diff
@@ -0,0 +1,335 @@
+		!CONFIG[Model8096L] {}
+		!CONFIG:HAS[!LinkB9PSModule[nozzle]] {}
+	}
+	@title = XLR81 Agena Engine Series
```
There's no reason to rename this

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Return_Capsules.cfg:
```diff
@@ -0,0 +1,199 @@
+			ignoreForIsp = True
+		}
+		atmosphereCurve
+		{
```
This is too high for an HTP thruster

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Strawman.cfg:
```diff
@@ -0,0 +1,153 @@
+	!MODULE[ModuleDataTransmitter],* {}
+	MODULE
+	{
+		name = ModuleRealAntenna
```
You should actually configure these as high gain antennas

Capkirk123 commented on GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg:
```diff
@@ -0,0 +1,797 @@
+	@title = Agena SLV3B Interstage
+	@manufacturer = Convair
+	@description = Half length 1.5m interstage for the Atlas SLV-3B rocket.
+	@mass = 0.1
```
Could you try to configure these a bit better instead of setting them all the same?

Capkirk123 reviewed (changes requested):
A good start, but could use some work. It looks like there wasn't much of an attempt to set masses for a lot of these parts.

You should look at the existing configs for these parts, either in ROCapsules or the old configs section, a lot of them were already configured pretty well.

Capkirk123 commented:
Oh, and following recent attempts to get rid of massless batteries, _don't use SM tanks for anything that shouldn't have a battery in it, especially if that tank is massless_

SkyPhoenix999 committed:
Update RO_Agena.cfg

Fixing everything asked for

Just 6 months late

SkyPhoenix999 committed:
Update RO_Agena_Engines.cfg

Fixing Stuff Yet Again

SkyPhoenix999 committed:
Update RO_Strawman.cfg

More Fixing

SkyPhoenix999 committed:
Moving the Waterfall Configs

Moving the Waterfall Configs to the Waterfall Folder

SkyPhoenix999 committed:
Delete GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Waterfall_Agena.cfg

Deleting the Original since the file was moved over to the Waterfall_Configs Folder

SkyPhoenix999 committed:
Update RO_Return_Capsules.cfg

Changing HTP RCS ISP

Capkirk123 committed:
Improve BDB Agena configs

SkyPhoenix999 committed:
Update RO_Agena.cfg

Fixed MFT Configs

Replaced Tank-Int-Al with Tank-Iso-Al as Tank-Int-Al does not exist.

SkyPhoenix999 committed:
Agena Engine Merge

- Merged XLR-81 Engines for the BDB update
- Removed the Engine only and RCS only variants of the SPS due to not working properly

SkyPhoenix999 committed:
Waterfall Update for Engine Merge

SkyPhoenix999 reviewed (commented):
Works in Game. Forgot I reviewed myself
